src/plopp/graphics/colormapper.py

```diff
@@ -194,6 +200,19 @@ def _set_artists_colors(self, key: str = None):
         for k in keys:
             self.artists[k].set_colors(self.rgba(self.artists[k].data))
 
+    def _set_normalizer_limits(self):
+        """
+        Synchronize the underlying normalizer limits to the current state.
+        """
+        # Note that the order matters here, as for a normalizer vmin cannot be set above
+        # the current vmax.
+        if self.vmin >= self.normalizer.vmax:
+            self.normalizer.vmax = self.vmax
+            self.normalizer.vmin = self.vmin
+        else:
+            self.normalizer.vmin = self.vmin
+            self.normalizer.vmax = self.vmax
```

Comment:
The two branches are identical. Why is there an `if`?

Do the setters for `vmin` and `vmax` do something on top of setting values?

Reply:
The comment just above about how the order matters tries to describe it.
Something happens behind the scenes, but I am not sure what.